public/app/routes/routes.tsx

```diff
@@ -153,7 +154,7 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "DashboardListPage"*/ 'app/features/search/components/DashboardListPage')
       ),
     },
-    {
+    config.exploreEnabled && {
```

Comment:
Just realized we do it differently for alerting and profile. Maybe we could do something similar, by creating a `FeatureTogglePage.tsx`:

```typescript
    {
      path: '/explore',
      pageClass: 'page-explore',
      roles: () =>
        contextSrv.evaluatePermission(
          () => (config.viewersCanEdit ? [] : ['Editor', 'Admin']),
          [AccessControlAction.DataSourcesExplore]
        ),
      component: config.exploreEnabled
        ? SafeDynamicImport(() => import(/* webpackChunkName: "explore" */ 'app/features/explore/Wrapper'))
        : SafeDynamicImport(
            () => import(/* webpackChunkName: "Explore feature toggle page"*/ 'app/features/explore/FeatureTogglePage')
          ),
    }
```

This way we could get a nice info page (something like below) instead of regular 404 (and avoid filtering at the bottom):

<img width="411" alt="Screen Shot 2022-04-13 at 19 03 44" src="https://user-images.githubusercontent.com/745532/163232980-3d172777-f044-4239-bf68-750c52c7b4a7.png">

Reply:
good point, modified to look like this:
![Screenshot 2022-04-14 at 11 33 37](https://user-images.githubusercontent.com/1170767/163370998-76c892c2-d6eb-442c-a4c0-021713f5727b.png)